src/devices/Dhtxx/DhtSensor.cs

```diff
@@ -51,7 +52,7 @@ private void Wait(double milliseconds)
         /// <remarks>
         /// If last read was not successfull, it returns double.MaxValue
         /// </remarks>
-        public double Temperature => (_dhtType == DhtType.Dht11) ? GetTempDht11() : GetTempDht22();
+        public float Temperature => (float)((_dhtType == DhtType.Dht11) ? GetTempDht11() : GetTempDht22());
```

Comment:
didn't want to make further refactoring before we decide on returned type. I think that float is enough since abstraction should be a common denominator and float's precision reflects that well IMO (we can always have IHighPrecisionTemperatureSensor there in the future which could take BigInteger decimal)

Reply:
Let's keep this as Double for now; as Tanner said offline, all Single values are cleanly representable by Double.

Reply:
fair enough - most temperature sensors I've seen were 16bit int scaled and shifted by some constant factor so was basing it on that. I don't feel strongly either way

Reply:
also the other reason for float is that Vector3 uses float internally and and Vector3 is what we use for Acceleration and other 3D vectors in this repo

Reply:
another alternative is to make ITemperature/IThermometer generic and not depend on any specific type

Reply:
I think we should stick with double for now as well and then if we ever see a scenario where float would work better then we can change this.

Reply:
fair enough, I honestly wish we'd have Vector3 with double internals 😄

Reply:
> fair enough, I honestly wish we'd have Vector3 with double internals

That would be https://github.com/dotnet/corefx/issues/25334, which is ready for review but will likely be post 3.0